LIMITATIONS.md

```diff
@@ -0,0 +1,22 @@
+# Limitations
+
+* <https://nmstate.atlassian.net/browse/NMSTATE-235>
+
+  The `libnmstate` does not differentiate `InterfaceState.DOWN` and
+  `InterfaceState.ABSENT`. They both remove the on-disk Network Manager
+  connection profile of specified interface.
+
+* <https://nmstate.atlassian.net/browse/NMSTATE-236>
+
+  All the network state applied by `libnmstate` is preserved after
+  reboot, except for disabling IPv6.
+
+* <https://nmstate.atlassian.net/browse/NMSTATE-115>
+
+  Due to the limitation of NetworkManager, Nmstate does not support disabling
+  DHCPv6 while enabling autoconf yet.
+
+* <https://nmstate.atlassian.net/browse/NMSTATE-212>
+
+  Currently, Nmstate requires the existence of interface section for all the
```

Comment:
Nmstate requires that the interfaces which appear in the IP routes as `next hop`, to be explicitly specified in the interface section.